### DISCUSSION TITLE
tensorboard does not show steps in audio

### DISCUSSION THREAD

**Discussion by madey83** [idiap/coqui-ai-TTS#162] [Q&A]
i do VITS trainig from scrach and when i would like to hear test sentens on  tensorboard  i got 0 in steps count even if my learning steps is 250.

this is also was the problem when i done fine-tuning...

![image](https://github.com/user-attachments/assets/2ed47597-1d20-4853-bb33-dbf8b291a082)

below my executed code:

import os
from glob import glob

from trainer import Trainer, TrainerArgs

from TTS.tts.configs.shared_configs import BaseDatasetConfig
from TTS.tts.configs.vits_config import VitsConfig
from TTS.tts.datasets import load_tts_samples
from TTS.tts.models.vits import CharactersConfig, Vits, VitsArgs, VitsAudioConfig
from TTS.tts.utils.languages import LanguageManager
from TTS.tts.utils.speakers import SpeakerManager
from TTS.tts.utils.text.tokenizer import TTSTokenizer
from TTS.utils.audio import AudioProcessor

def main():

	#output_path = os.path.dirname(os.path.abspath(__file__))
	output_path = "D:\\TTS-Training\\Vits-PL-Model-training\\Output"
	dataset_config = BaseDatasetConfig(
		formatter="ljspeech", meta_file_train="metadata.csv", 
        #path="D:\\TTS-Training\\Vits-PL-Model-training\\dataLJSpeech_final"
        path="D:\\TTS-Training\\LJSpeech-1.1"
	)
	audio_config = VitsAudioConfig(
		sample_rate=22050, win_length=1024, hop_length=256, num_mels=80, mel_fmin=0, mel_fmax=None
	)
        
	config = VitsConfig(
		audio=audio_config,
		run_name="Vits-TTS-PL",
		batch_size=16,
		eval_batch_size=8,
		batch_group_size=5,
		num_loader_workers=4,
		num_eval_loader_workers=2,
		run_eval=True,
		test_delay_epochs=-1,
		epochs=1000,
        save_step=1000,
        save_checkpoints=True,
        save_n_checkpoints=2,
        save_best_after=5000,
        add_blank=True,                      
		text_cleaner="multilingual_cleaners",
		use_phonemes=True,
		phoneme_language="pl",
		phoneme_cache_path=os.path.join(output_path, "phoneme_cache"),
		compute_input_seq_cache=True,
		print_step=25,
		print_eval=True,
		mixed_precision=False,
		output_path=output_path,
		datasets=[dataset_config],
		cudnn_benchmark=False,
		test_sentences=[
            "Nazywam się Rusty Sabich i jestem prokuratorem w tym procesie.",
            "Jeśli uznacie to za bardzo prawdopodobne, musicie wypuścić LEDU 20 października 1963 roku.",
		],
	)

	# INITIALIZE THE AUDIO PROCESSOR
	# Audio processor is used for feature extraction and audio I/O.
	# It mainly serves to the dataloader and the training loggers.
	ap = AudioProcessor.init_from_config(config)

	# INITIALIZE THE TOKENIZER
	# Tokenizer is used to convert text to sequences of token IDs.
	# config is updated with the default characters if not defined in the config.
	tokenizer, config = TTSTokenizer.init_from_config(config)

	# LOAD DATA SAMPLES
	# Each sample is a list of ```[text, audio_file_path, speaker_name]```
	# You can define your custom sample loader returning the list of samples.
	# Or define your custom formatter and pass it to the `load_tts_samples`.
	# Check `TTS.tts.datasets.load_tts_samples` for more details.
	train_samples, eval_samples = load_tts_samples(
		dataset_config,
		eval_split=True,
		eval_split_max_size=config.eval_split_max_size,
		eval_split_size=config.eval_split_size,
	)

	# init model
	model = Vits(config, ap, tokenizer, speaker_manager=None)

	# init the trainer and 🚀
	trainer = Trainer(
		TrainerArgs(),
		config,
		output_path,
		model=model,
		train_samples=train_samples,
		eval_samples=eval_samples,
	)
	trainer.fit()
	print("Complete!")

from multiprocessing import Process, freeze_support
if __name__ == '__main__':
    freeze_support()  # needed for Windows
    main()

any suggestion

**Comment by eginhard** [comment]
Let it run for some time, it should show more outputs eventually and you can scroll through different steps then.

**Comment by madey83** [comment]
thanks for answer, but first time when i star traing i have got from fist step audio available... 

when i have reinstall windows this does not show any more even if i move to 1000 steps...
